pagefaultgames/pokerogue
# Localized game-mode and save-slot-select-ui-handler

## What are the changes?

Localized game-mode and save-slot-select-ui-handler and added ptBR translations
## Why am I doing these changes?



Those pages weren't localized yet
## What did change?


Translations and text in game-mode and save-slot-select-ui-handler files
"Empty" locale was only used in the save-slot-select-ui-handler, so I moved it from menu.ts to save-slot-select-ui-handler.ts
### Screenshots/Videos (all screenshots are in ptBR)


## Game Modes
### Before
![image](https://github.com/pagefaultgames/pokerogue/assets/79599290/3d5c78a0-5fdb-4e0f-b978-7cb301e6989d)
### After
![image](https://github.com/pagefaultgames/pokerogue/assets/79599290/286611f9-9a0d-44f9-b9dc-0d355819a4d2)

## Save Slots
### Before
![image](https://github.com/pagefaultgames/pokerogue/assets/79599290/82fdc3a5-ba75-4d03-813a-4f63ae7ffeaa)
![image](https://github.com/pagefaultgames/pokerogue/assets/79599290/78e4163e-fb22-4ce9-96d3-d8a7727f6269)
### After
![image](https://github.com/pagefaultgames/pokerogue/assets/79599290/4cd5894b-4382-4e7a-9b6e-4dc22a65aee6)
![image](https://github.com/pagefaultgames/pokerogue/assets/79599290/d82a40ae-6d69-4d8a-8247-b7745bb08a14)

## Overwrite Data
### Before
![image](https://github.com/pagefaultgames/pokerogue/assets/79599290/dd2ee67c-64f2-4572-932d-cd0cbb738883)
### After
![image](https://github.com/pagefaultgames/pokerogue/assets/79599290/320af9ba-a5d4-4fee-bf3e-286c773438a9)

## How to test the changes?




Check the new game screen, load game screen and the text when overwriting a save slot.
## Checklist
Overlap with #1674, but he just localized the overwriteData text
